corehq/apps/data_dictionary/tests/test_views.py

```diff
@@ -1,19 +1,32 @@
 import json
+import random
+import string
 import uuid
 from unittest.mock import patch
 
 from django.test import Client, TestCase
+from django.urls import re_path as url
 from django.urls import reverse
 
-from corehq.apps.data_dictionary.models import CaseProperty, CasePropertyGroup, CasePropertyAllowedValue, CaseType
+from corehq import privileges
+from corehq.apps.data_dictionary.models import (
+    CaseProperty,
+    CasePropertyAllowedValue,
+    CasePropertyGroup,
+    CaseType,
+)
+from corehq.apps.data_dictionary.urls import urlpatterns
+from corehq.apps.data_dictionary.views import data_dictionary_json_v2
 from corehq.apps.domain.shortcuts import create_domain
 from corehq.apps.geospatial.const import GPS_POINT_CASE_PROPERTY
-from corehq.apps.users.models import WebUser, HqPermissions
+from corehq.apps.users.models import HqPermissions, WebUser
 from corehq.apps.users.models_role import UserRole
+from corehq.util.test_utils import flag_enabled, privilege_enabled
 
-from corehq.util.test_utils import privilege_enabled
-from corehq import privileges
-from corehq.util.test_utils import flag_enabled
+# TODO Remove this once we migrate to the new view
+urlpatterns.insert(0, url(r"^json_v2/$", data_dictionary_json_v2, name='data_dictionary_json_v2'))
```

Comment:
Instead of dynamically inserting these URL patterns for the tests, would it make sense to rather just add these to the URLs file? They already have a different URL path to the current DD JSON endpoint, and we can always switch them out when we remove the old views.

Reply:
Sure, that can be done. I only was not in favour of adding new urls , however considering we are going to remove them in short time, I am good to add these in the urls.

Reply:
Addressed in 384263b3d074eaa824895851e1f7bdcb4e6b0e94